packages/shared/components/popups/PrivacyPolicyUpdate.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="typescript">
+    import { onDestroy } from 'svelte'
+    import { Button, Checkbox, Text } from 'shared/components'
+    import { lastAcceptedPrivacyPolicy } from 'shared/lib/appSettings'
+    import { localize } from 'shared/lib/i18n'
+    import { closePopup } from 'shared/lib/popup'
+    import { PRIVACY_POLICY_VERSION } from 'shared/lib/app'
+    import { Electron } from 'shared/lib/electron';
+
+    let checked = false
+    const PRIVACY_POLICY_LINK = 'https://firefly.iota.org/privacy'
+
+    const handleViewClick = () => {
+        Electron.openUrl(PRIVACY_POLICY_LINK)
+    }
+
+    const handleConfirmClick = () => {
+        lastAcceptedPrivacyPolicy.set(PRIVACY_POLICY_VERSION)
+
+        closePopup(true)
+    }
+
+    onDestroy(() => closePopup)
```

Comment:
Does the call to `closePopup` inside of `onDestroy` need to be passed `true` to force close it? It seems to be opened with `preventClose`.

Reply:
Ooh good catch

Reply:
Just realized we don't usually use `onDestroy` for the popups and the only way this component would be destroyed is if the user clicked "confirm", so I'll remove the `onDestroy`